db/warehouse/migrate/20240829152828_add_to_form_submission.rb

```diff
@@ -0,0 +1,14 @@
+#  Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+#  License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+#
+
+class AddToFormSubmission < ActiveRecord::Migration[7.0]
+  def change
+    add_column :hmis_external_form_submissions, :cleaned_values, :jsonb, null: true
+
+    safety_assured do
+      add_reference :hmis_external_form_submissions, :enrollment, index: true, null: true
```

Comment:
I added this with add_reference but without explicitly creating a foreign key because I couldn't get it to work correctly with this [truncation](https://github.com/greenriver/hmis-warehouse/blob/ad0735ad768cb9e9018f6088e23f2ca23036bba0/app/models/grda_warehouse/utility.rb#L4). I remember running into this before (with user staff assignment) but the resolution was a bit different because the relationship spanned databases, so just wanted to flag it and check that it looks okay.